src/network/serverpackethandler.cpp

```diff
@@ -478,11 +478,14 @@ void Server::process_PlayerPos(RemotePlayer *player, PlayerSAO *playersao,
 
 	f32 fov = 0;
 	u8 wanted_range = 0;
+	u8 bits; // bits instead of bool so it is extensible later
```

Comment:
```suggestion
	u8 bits = 0; // bits instead of bool so it is extensible later
```

Reply:
That should be the default, no? But can't hurt to add it.

Reply:
No, default init in c++ is an indeterminate value in this case. See <https://en.cppreference.com/w/cpp/language/default_initialization>.
You only get 0 through value init (`u8 bits{};`).